fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -130,4 +160,17 @@ public String apply(final Integer type) {
 
     private static final Logger LOGGER = getLogger(DefaultMessageFactory.class);
 
+    private String getPropertiesAsString(final FixityEvent fixityEvent) {
+        final StringBuilder propsStrBuilder = new StringBuilder();
+        for (Map.Entry<String,String> entry : fixityEvent.getInfo().entrySet()) {
+            propsStrBuilder.append(entry.getKey()).append("->").append(entry.getValue()).append(",");
```

Comment:
We should discuss what we expect a fixity message to contain.
I would prefer name/value pairs where the name is the `header field` and the value is a simple literal, like `SUCCESS`.
Embedding data structures in the message headers seems fragile and less than ideal.
@acoburn's suggestion of:
`org.fcrepo.jms.fixity` = `SUCCESS`
seems reasonable.